apis/resources/v1alpha1/bentorequest_types.go

```diff
@@ -88,6 +88,8 @@ type BentoRequestSpec struct {
 
 	// +kubebuilder:validation:Optional
 	DownloaderContainerEnvFrom []corev1.EnvFromSource `json:"downloaderContainerEnvFrom,omitempty"`
+	// +kubebuilder:validation:Optional
+	DownloaderContainerEnv []corev1.EnvVar `json:"downloaderContainerEnv,omitempty"`
```

Comment:
Is `DownloaderContainerEnv` still necessary since we already have a `DownloaderContainerEnvFrom`?

Reply:
i discussed with @FogDong. Seems the more natural way is to have an EnvVar instead of EnvFromSource. Since we are reading the plain-text env vars from bento.yaml

Reply:
@rickzx However, passing tokens in plain text using EnvVar is a very insecure practice, which is why DownloaderContainer previously only provided EnvFrom and not Env.